legend-engine-pure-code-compiled-core/src/main/resources/core/pure/executionPlan/platformBinding/typeInfo/typeInfo.pure

```diff
@@ -649,3 +656,89 @@ function  <<access.private>> meta::pure::executionPlan::platformBinding::typeInf
 {
    {pe1:PackageableElement[1], pe2:PackageableElement[1] | $pe1->elementToPath() == $pe2->elementToPath()};
 }
+
+function <<access.private>> meta::pure::executionPlan::platformBinding::typeInfo::typeInfoFromTreeExtra(info:TypeInfoSet[1],tree:GraphFetchTree[1], debug:DebugContext[1]): TypeInfoSet[1]
+{
+  print(if($debug.debug,|$debug.space+'getting extra types from tree : '+  + $tree->asString(false) + '\n',|''));
+                                    
+  let extraTypes= $tree->match([
+    p:RoutedPropertyGraphFetchTree[1]| 
+             $p.propertyMapping->map(pm|  
+                  $pm->match([
+                        ppm: meta::pure::mapping::modelToModel::PurePropertyMapping[1] | 
+                            $ppm.transform->evaluateAndDeactivate()->scanTypesFromAny([], $debug->indent()->indent());,
```

Comment:
transform is a Lambda. Is evaulateAndDeactivate required for that?

Reply:
without this, an error is raised for some variables not found in context